Graph and Linked List Practice Game/Assets/Scripts/GraphNode.cs

```diff
@@ -0,0 +1,158 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GraphSearching
```

Comment:
Per Mantan, this is copy paste for providing context of his own implementation, let's remove it if we're not going to use it.